LICENSE.md

```diff
@@ -1,4 +1,4 @@
-OpenStudio(R), Copyright (c) 2020-2023, OpenStudio Coalition and other contributors. All rights reserved.
+OpenStudio(R), Copyright (c) 2020-2024, OpenStudio Coalition and other contributors. All rights reserved.
```

Comment:
Update the root LICENSE.md. This is the **ONLY** file we need to keep up to date now! :confetti_ball: